### DISCUSSION TITLE
django.core.exceptions.ImproperlyConfigured: Error loading MySQLdb module. | fresh gethue/hue:latest

### DISCUSSION THREAD

**Discussion by Basserado** [cloudera/hue#2774] [Questions] [answered]
I just want to start a fresh docker-compose instance of hue, but I'm getting the following error:

`django.core.exceptions.ImproperlyConfigured: Error loading MySQLdb module.
Did you install mysqlclient?
Traceback (most recent call last):
  File "/usr/share/hue/build/env/lib/python3.8/site-packages/django/db/backends/mysql/base.py", line 15, in <module>
    import MySQLdb as Database
ModuleNotFoundError: No module named 'MySQLdb'`

That happen since the latest image of gethue/hue:latest. Is it an image problem?

**Comment by Harshg999** [comment]
Hi @Basserado, most probably image is missing the `mysqlclient` package. Can you install it explicitly and see if that works for you?

**Comment by gitaramK** [comment]
I am also facing exact same issue

```
ModuleNotFoundError: No module named 'MySQLdb'
hue             |
hue             | The above exception was the direct cause of the following exception:
hue             |
hue             | Traceback (most recent call last):
hue             |   File "/usr/share/hue/build/env/bin/hue", line 33, in <module>
```

**Comment by nikosheng** [comment]
I have created a [PR](https://github.com/cloudera/hue/pull/2762) and fixed the problem

**Comment by Harshg999** [comment, marked as answer]
We have removed the `mysqlclient` package from our requirements.txt because of licence issues.
`mysqlclient` is GPL licensed vs Hue which is Apache licensed.

Users can add the package explicitly if they want to have MySQL.

For Hue's database, you can alternatively use other database like postgres! 
https://docs.gethue.com/administrator/installation/dependencies/#postgresql
https://docs.gethue.com/administrator/installation/dependencies/#macos (refer postgres steps)

**Reply by nikosheng** [reply]
Thanks for your update and notification
